src/plugins/home/public/application/components/guided_onboarding/constants.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const i18nTexts = {
```

Comment:
nit: colocate these strings with the actual component(s), outside the component function.  It can be hard to know if a string is still relevant, and having them in a remote singleton can be confusing.